command.go

```diff
@@ -269,13 +271,14 @@ func ExecuteSetLineHeight(c Command, v *VHS) {
 
 // ExecuteSetTheme applies the theme on the vhs.
 func ExecuteSetTheme(c Command, v *VHS) {
-	err := json.Unmarshal([]byte(c.Args), &v.Options.Theme)
+	var err error
+	v.Options.Theme, err = getTheme(c.Args)
```

Comment:
The new `vhs.Errors` might be useful here. We should probably fail fast and not generate the full GIF if the theme will be wrong.